pages/index.html

```diff
@@ -4,9 +4,37 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Michelle's Portfolio Website</title>
+  <link href="/styles/style.css" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/3e8fd03e4e.js"
+  crossorigin="anonymous"></script>
 </head>
+
 <body>
-  
+<content id="main">
+<header>
+  <iframe width="0" height="0" src="https://www.youtube.com/embed/Atp8GPADPDM?controls=0?rel=0&autoplay=1" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<nav>
+  <h1>Welcome to Michelle's Portfolio!</h1>
+  <ul class="top-bottom-links">
+    <li>Main</li>
+    <li><a href="/pages/about.html">About</a></li>
+    <li><a href="/pages/portfolio.html">Portfolio</a></li>
+  </ul>
+</nav>
+</header>
+<section id="main">
+</section> 
+<footer>
+  <nav>
+    <ul class="top-bottom-links">
+      <li><a href="https://calendly.com/michellemorales" target="_blank"><i class="far fa-calendar-check"></i></a></li>
+      <li><a href="https://www.linkedin.com/in/moralesmi/" target="_blank"><i class="fab fa-linkedin"></i></a></li>
+      <li><a href="https://github.com/DynamicDev3000?tab=repositories" target="_blank"><i class="fab fa-github"></i></a</li>
```

Comment:
There's a missing `>` in the `</a>` here, that looks like it got copy/pasted into the other pages as well.

Running your html through a validator would catch this.